src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java

```diff
@@ -67,9 +67,9 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         predicate = new NameContainsKeywordsPredicate(Arrays.asList("Carol"));
         assertFalse(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
 
-        // Keywords match phone and email, but does not match name
+        // Keywords match telegram, email and address, but does not match name
         predicate = new NameContainsKeywordsPredicate(Arrays.asList("12345", "alice@email.com", "Main", "Street"));
```

Comment:
I don't think there should be address?
```suggestion
        // Keywords match telegram and email, but does not match name
        predicate = new NameContainsKeywordsPredicate(Arrays.asList("12345", "alice@email.com"));
```